idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -789,3 +889,18 @@ def default_material_balance_type(blk):
 
     def default_energy_balance_type(blk):
         return EnergyBalanceType.enthalpyTotal
+
+    def calculate_scaling_factors(self):
+        super().calculate_scaling_factors()
+
+        if hasattr(self, "sum_component_eqn"):
+            for t, v in self.sum_component_eqn.items():
+                iscale.set_scaling_factor(v, 1e2)
+
+        if hasattr(self._k_rxn, "visc_d_constraint"):
+            for t, v in self.visc_d_constraint.items():
+                iscale.set_scaling_factor(v, 1e6)
+
+        if hasattr(self._k_rxn, "therm_cond_constraint"):
```

Comment:
What does _k_rxn represent?

Reply:
That is a typing error, I will remove the extra attributes on those lines.